Hazel/src/Hazel/Scene/SceneSerializer.cpp

```diff
@@ -88,6 +88,10 @@ namespace YAML {
 }
 namespace Hazel {
 
+	static void SerializeAllEntityComponents(YAML::Emitter& out, Entity entity);
+
+	static void DeseralizeAllEntryComponents(YAML::detail::iterator_value& entity, Entity& deserializedEntity);
```

Comment:
Hi, you have typo in every DeseralizeAllEntryComponents it should be DeserializeAllEntryComponents

Reply:
Good catch 👍 I will fix when I get home 🏡

Reply:
Fixed